server/src/test/scala/org/scassandra/server/cors/CorsSupportTest.scala

```diff
@@ -0,0 +1,58 @@
+package org.scassandra.server.cors
+
+import com.typesafe.scalalogging.LazyLogging
+import org.scalatest._
+import org.scassandra.server.ScassandraConfig
+import org.scassandra.server.priming.cors.CorsSupport
+import spray.http._
+import spray.routing.HttpService
+import spray.testkit.ScalatestRouteTest
+
+
+class CorsSupportTest extends FlatSpec
+  with ScalatestRouteTest
+  with HttpService
+  with LazyLogging
+  with CorsSupport
+  with Matchers {
+  def actorRefFactory = system
+
+  val testRoute = path("test") {
```

Comment:
I can't really find an obvious way to modify the config so we can have a test without ScassandraConfig.corsEnabled,  I tried this out manually with a browser and xhr and it worked.  I think its pretty straightforward so we can get away without having an explicit test here.